MaxText/inference_microbenchmark.py

```diff
@@ -27,10 +28,29 @@
 import maxtext_utils
 import pyconfig
 
-
 _WARMUP_ITERS = 2
 
 
+def debug_kv_cache(kv_cache):
+  singler_kv_cache = kv_cache["cache"]["decoder"]["layers_0"]["self_attention"]["AttentionOp_0"]
```

Comment:
Nit: Is this supposed to be "single" or something else?

Reply:
added single layer